pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -189,6 +191,7 @@ var _ = Describe("Schema", func() {
 		`<memballoon model="virtio">
       <stats period="10"></stats>
     </memballoon>`)
+	exampleXMLwithDeviceWithoutUserAliasPrefix = strings.Replace(exampleXML, "ua-tablet0", "tablet0", 1)
```

Comment:
Looks like you can keep this completely in the test. I don't think you need to have a global function or do something in BeforeEach. Just prepare it in `It`.